api/src/pcapi/routes/backoffice/collective_offers/collective_offers_blueprint.py

```diff
@@ -648,28 +651,23 @@ def _batch_validate_or_reject_collective_offers(
             if validation is offer_mixin.OfferValidationStatus.APPROVED and collective_offer.institutionId is not None:
                 try:
                     adage_client.notify_institution_association(serialize_collective_offer(collective_offer))
-                except educational_exceptions.AdageInvalidEmailException:
-                    # in the case of an invalid institution email, adage is not notified but we still want to validate of reject the offer
-                    flash(
-                        Markup("Email invalide pour l'offre <b>{offer_id}</b>, ADAGE n'a pas été notifié").format(
-                            offer_id=collective_offer.id
-                        ),
-                        "warning",
-                    )
                 except educational_exceptions.AdageException as exp:
-                    flash(
-                        Markup(
-                            "Erreur lors de la notification à ADAGE pour l'offre <b>{offer_id}</b> : {message}"
-                        ).format(offer_id=collective_offer.id, message=exp.message),
-                        "warning",
+                    # when the call to Adage fails, we log an error and warn the user but we still validate the offer
+                    logger.exception(
+                        "Error on Adage notify when validating collective offer",
+                        extra={"collective_offer_id": collective_offer_id, "status_code": exp.status_code},
```

Comment:
```suggestion
                        extra={"collective_offer_id": collective_offer_id, "status_code": exp.status_code, "exp": exp},
```
Le status code pour regrouper + la repr de l'exception pour avoir le verbose